library/src/main/java/com/voysis/wakeword/Utils.kt

```diff
@@ -0,0 +1,22 @@
+package com.voysis.wakeword
+
+import org.apache.commons.collections.buffer.CircularFifoBuffer
+import org.tensorflow.lite.Interpreter
+
+fun processWakeword(buffer: CircularFifoBuffer, interpreter: Interpreter): Float {
+    val input = buffer.toArray().map { it as Float }.toFloatArray()
+    val output = Array(1) { FloatArray(2) }
+    interpreter.run(input, output)
+    return output[0][1]
+}
+
+fun isAboveThreshold(input: Float, threshold: Float): Int = if (input >= threshold) {
```

Comment:
Very minor but you can change this to:
```suggestion
fun isAboveThreshold(input: Float, threshold: Float): Int = if (input >= threshold) 1 else 0
```
It's up to your own preference